demos/loan-platform/setup.sh

```diff
@@ -0,0 +1,262 @@
+if [[ $? -ne 0 ]] 
+then
+  err "failed to modify terraform main.tf file for fnman smart contract arn."
+  exit 1
+fi
+
+# <css-vendia-smart-contract-arn>
+sed -i '' -e "s|<css-vendia-smart-contract-arn>|${smart_contract_arn[2]}|g" ./src/terraform/main.tf
+if [[ $? -ne 0 ]] 
+then
+  err "failed to modify terraform main.tf file for css smart contract arn."
+  exit 1
+fi
+
+
+(cd src/terraform; terraform init; terraform apply -auto-approve)
+
+# 4. Share smart contracts created dependent on step 3)
+
+# Get all lambda arns to create .share.env file
+delinquent_lambda_arn=$(cd src/terraform; terraform output --raw lambda_delinquent_output)
+upb_lambda_arn=$(cd src/terraform; terraform output --raw lambda_upb_output)
+wair_lambda_arn=$(cd src/terraform; terraform output --raw lambda_wair_output)
+
+### Generate .share.env file
+### This will not work if in the future order of the returned nodes changes.
+
+echo -e > ./src/.share.env \
+"COOP_GQL_URL=${graphqlAPI[1]}
+COOP_GQL_APIKEY=${node_keys[1]}
+CSS_GQL_URL=${graphqlAPI[2]}
+CSS_GQL_APIKEY=${node_keys[2]}
+FNMA_GQL_URL=${graphqlAPI[3]}
+FNMA_GQL_APIKEY=${node_keys[3]}
+JPMC_GQL_URL=${graphqlAPI[4]}
+JPMC_GQL_APIKEY=${node_keys[4]}
+PHH_GQL_URL=${graphqlAPI[5]}
+PHH_GQL_APIKEY=${node_keys[5]}
+UPB_LAMBDA_ARN=${upb_lambda_arn}
+DELINQUENT_LAMBDA_ARN=${delinquent_lambda_arn}
+WAIR_LAMBDA_ARN=${wair_lambda_arn}"
+
+
+# Run npm to create all the smart contracts
+
+(cd src; npm i; npm run createUpbSmartContract;)
+if [[ $? -ne 0 ]] 
+then
+  err "failed to create upb smart contract."
+  exit 1
+fi
+
+
+(cd src; npm run createDelinquentSmartContract;)
+if [[ $? -ne 0 ]] 
+then
+  err "failed to create delingquent smart contract."
+  exit 1
+fi
+
+(cd src; npm run createWairSmartContract;)
+if [[ $? -ne 0 ]] 
+then
+  err "failed to create wair smart contract."
+  exit 1
+fi
+
+## Create empty folders for sample data
+
+mkdir data
+mkdir data/payments
+mkdir data/loans
+
+(cd src; npm run generatePayments;)
+(cd src; npm run generateLoans;)
+
+echo "All resources are created!"
+
+# 5. Run Front end app
+
+echo -e > ./src/app/src/__config.tsx \
+"import { ClientOptions } from '@vendia/client';
+
+type Config = Record<string, ClientOptions>;
+
+export const config: Config = {
+  FNMANode: {
+    apiUrl: '${graphqlAPI[3]}',
+    websocketUrl:
+      '${websocket_url[3]}',
+    apiKey: '${node_keys[3]}',
+    debug: true,
+  },
+  COOPServicingNode: {
+    apiUrl: '${graphqlAPI[1]}',
+    websocketUrl:
+      '${websocket_url[1]}',
+    apiKey: '${node_keys[1]}',
+    debug: true,
+  },
+  PHHServicingNode: {
+    apiUrl: '${graphqlAPI[5]}',
+    websocketUrl:
+      '${websocket_url[5]}',
+    apiKey: '${node_keys[5]}',
+    debug: true,
+  },
+};
+
+export const nodes = Object.keys(config);"
+
+if [[ $? -ne 0 ]] 
+then
+  err "failed to create __config.tsx file."
+  exit 1
+fi
+
+## install vendia module and dependencies
+(cd src/app; share client:pull;)
```

Comment:
Could you update this command to provide the uni and node to generate the SDK for?

Reply:
Done. The new command is 
```
(cd src/app; printf "y" | share client:pull --uni $uni_name --node $nodes_names[4];)
```